app/assets/stylesheets/osem.css.scss

```diff
@@ -56,3 +56,15 @@ fieldset {
 }
 
 .bootstrap-switch { height: 1.7em }
+
+.comment-reply{
+  padding-top: 40px;
+}
+
+p.comment-body {
+  padding-top: 20px;
+}
+
+.well.comment-section {
+  padding-bottom: 40px;
+}
```

Comment:
Please insert a new blank line at the end of files. It's a styling thing and I don't know why rubocop is not complaining.
(https://github.com/openSUSE/osem/blob/master/.rubocop.yml#L117)